src/main/java/com/maddyhome/idea/vim/extension/argtextobj/VimArgTextObjExtension.java

```diff
@@ -298,33 +298,52 @@ private static class ArgBoundsFinder {
      * @param position starting position.
      */
     boolean findBoundsAt(int position) throws IllegalStateException {
-      if (text.isEmpty()) {
+      if (text.length() == 0) {
```

Comment:
Isn't `isEmpty()` the same, but more readable option?

Reply:
Good point. Fixed.